google-oauth-client/src/main/java/com/google/api/client/auth/openidconnect/IdTokenVerifier.java

```diff
@@ -242,11 +247,7 @@ public boolean verify(IdToken idToken) {
     try {
       return verifySignature(idToken);
     } catch (VerificationException ex) {
-      LOGGER.log(
-          Level.SEVERE,
-          "id token signature verification failed. "
-              + "Please see docs for IdTokenVerifier for default settings and configuration options",
-          ex);
+      LOGGER.log(Level.INFO, "Id token signature verification failed. ", ex);
```

Comment:
shouldn't log level be higher?

Reply:
Previously it was higher because we swallowed the potential public key fetch error problem... 

Now it is part of normal operation.